File: 01. Source Code/02. Android/SwipeSafe/app/src/main/java/nts/swipesafe/model/MobileCommentResultModel.java
package nts.swipesafe.model;

import java.util.List;

public class MobileCommentResultModel {
    /// <summary>
    /// Id phản hồi
    /// </summary>
    public long id ;

    /// <summary>
    /// Id phản hồi con
    /// </summary>
    public Long parentCommentId;

    /// <summary>
    /// Nội dung phản hồi
    /// </summary>
    public String content;

    /// <summary>
    /// Id người phản hồi
    /// </summary>
    public String objectId ;

    /// <summary>
    /// Tên người phản hồi
    /// </summary>
    public String userName;

    /// <summary>
    /// Đường dẫn ảnh
    /// </summary>
    public String imagePath;

    /// <summary>
    /// 0: Chưa phê duyệt
    /// 1: Phê duyệt hiển thị ra bên ngoài
    /// 2: Xóa bình luận
    /// </summary>
    public int status ;

    /// <summary>
    /// Ngày tạo phản hồi
    /// </summary>
    public String commentDate ;

    public List<MobileCommentResultModel> listReply ;
}

File: 01. Source Code/01. Application/Elearning_Api/Elearning.Models/Models/Mobile/MobileComment/MobileCommentResultModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Elearning.Model.Models.Mobile.MobileComment
{
    public class MobileCommentResultModel
    {
        /// <summary>
        /// Id phản hồi
        /// </summary>
        public long Id { get; set; }

        /// <summary>
        /// Id phản hồi con
        /// </summary>
        public long? ParentCommentId { get; set; }

        /// <summary>
        /// Nội dung phản hồi
        /// </summary>
        public string Content { get; set; }

        /// <summary>
        /// Tên người phản hồi
        /// </summary>
        public string UserName { get; set; }

        /// <summary>
        /// Đường dẫn ảnh
        /// </summary>
        public string ImagePath { get; set; }

        /// <summary>
        /// 0: Chưa phê duyệt
        /// 1: Phê duyệt hiển thị ra bên ngoài
        /// 2: Xóa bình luận
        /// </summary>
        public int Status { get; set; }

        /// <summary>
        /// Ngày tạo phản hồi
        /// </summary>
        public DateTime CommentDate { get; set; }

        public List<MobileCommentResultModel> ListReply { get; set; }
        public MobileCommentResultModel()
        {
            ListReply = new List<MobileCommentResultModel>();
        }
    }
}
